addons/medical/functions/fnc_treatment.sqf

```diff
@@ -201,6 +201,12 @@ _treatmentTime = if (isNumber (_config >> "treatmentTime")) then {
 };
 
 // Start treatment
+if (_caller != ACE_player) exitWith {
+    // It's an AI that is doing the treatment
+    sleep _treatmentTime;
+    [[_caller, _target, _selectionName, _className, _items, _usersOfItems]] call DFUNC(treatment_success);
```

Comment:
Then I guess https://github.com/ArcticSnowSky/ACE3/commit/06a3d46ac77beb62afd598717557868482ed9e1b would be an acceptable solution.
The question now would be more over if it should be the same method, but with an additional parameter (for the caller) and that it will check autonom for caller == player then add UI stuff?